### PR TITLE
[STRATCONN-3376] Add phone number to TikTok Audiences

### PR DESCRIPTION
**Pull request by maryamsharif**
This PR maps and sends phone number to TikTok.

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
Screenshot 2023-11-16 at 1 32 32 PM
Screenshot 2023-11-16 at 1 34 04 PM

- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/tiktok-audiences/properties.ts [hunk 1]
```diff
@@ -56,11 +59,30 @@ export const send_email: InputField = {
...
+    '@if': {
+      exists: { '@path': '$.context.traits.phone' },
+      then: { '@path': '$.context.traits.phone' },
+      else: { '@path': '$.properties.phone' }
+    }
+  }
+}
+
+export const send_phone: InputField = {
+  label: 'Send Phone Number',
```
Comment (nick-Ag) [inline packages/destination-actions/src/destinations/tiktok-audiences/properties.ts:76]:
```suggestion
  label: 'Send Phone Number?',
```

**Review by nick-Ag — APPROVED** [review]
Looking good 🚀

**Comment by joe-ayoub-segment** [comment]
Will push and deploy on 28th

**Comment by marinhero** [comment]
This is a perfect candidate for a Hadron experiment. Can you please run one and we can discuss the results in #Hadron-Testing ?

**Comment by joe-ayoub-segment** [comment]
Hi @marinhero and @maryamsharif - confirming that I'll deploy on Wednesday given the "Ready for Release" label is applied. If you think you want to do Hadron or other further testing or if you don't want this deployed please remove the label :).

**Comment by maryamsharif** [comment]
Hi @joe-ayoub-segment, sorry for the delay. I ran a [hadron experiment](https://segment.datadoghq.com/dashboard/wie-q52-n97/hadron?refresh_mode=sliding&tpl_var_env=production&tpl_var_experiment=2ypzsyjyswt7up1vyyjwbm7avfs&from_ts=1701210291678&to_ts=1701224691678&live=true) and there were no diffs so we are good to deploy tomorrow! Thanks!

**Comment by joe-ayoub-segment** [comment]
Hi @maryamsharif this PR has been deployed

**Pull request merged**